docusaurus/docs-internals/editor-controls/multivalue-controls/transform.md

```diff
@@ -0,0 +1,197 @@
+
+Example of the transform:
+
+![Transform](/img/multivalue-controls/transform.png)
+
+### Parameters
+
+| Name               | Type            |    Default    | Description                                                             |
+|:-------------------|:----------------|:-------------:|:------------------------------------------------------------------------|
+| `id`               | `string`        |       -       | The identifier of the key where the transform will save your data       |
+| `type`             | `string`        |       -       | Type should be `"transform"` to use this control                        |
+| `label?`           | `string`        |       -       | The label displayed on the left side of the control                     |
+| `position?`        | `number`        |       -       | The position of the control in toolbar                                  |
+| `devices?`         | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled?`        | `boolean`       |    `false`    | Configure the condition under which the control is disabled or enabled. |
+| `config?.disabled` | `Array<Effect>` | `[ ]` | An array of animation names that should be disabled. They will not disappear from the list, but they cannot be selected <br/> <br/> <b>Effect = `"vertical"` \| `"horizontal"` \| `"transparency"` \| `"blur"` \| `"rotate"` \| `"scale"` \| `"mouseTrack"` \| `"mouseTilt"`</b> |
+
+### Return value
+
+Returns an object with the following values:
+
+```js
+{
+    "transformRotateDegree": undefined | string,
+    "transformRotateEnabled": undefined | boolean,
+    "transformRotateRotate": undefined | number,
+    "transformRotateRotate3D": undefined | string,
+    "transformRotateRotateX": undefined | number,
+    "transformRotateRotateY": undefined | number,
+    "transformRotateRotatePerspective": undefined | number,
+
+    "transformOffsetEnabled": undefined | boolean,
+    "transformOffsetOffsetX": undefined | number,
+    "transformOffsetOffsetY": undefined | number,
+
+    "transformSkewEnabled": undefined | boolean,
+    "transformSkewSkewX": undefined | number,
+    "transformSkewSkewY": undefined | number,
+
+    "transformScaleEnabled": undefined | boolean,
+    "transformScaleScaleX": undefined | number,
+    "transformScaleScaleY": undefined | number,
+    "transformScaleScaleXY": undefined | number,
+    "transformScaleScalePreserveSize": undefined | boolean,
+
+    "transformFlipEnabled": undefined | boolean,
+    "transformFlipFlipHorizontal": undefined | boolean,
+    "transformFlipFlipVertical": undefined | boolean,
+
+    "transformAnchorPointX": undefined | string,
+    "transformAnchorPointY": undefined | string
+}
```

Comment:
How look your return value: 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/0310376f-2acf-4e24-9d43-754f606726a7)

Make your return value and return value description look like in this PR: https://github.com/EasyBrizy/Brizy-Local-Editor/pull/259/files#diff-1f4f17d8adb93cf89a4dbfa84cd9eb9668f165d122ffefaf231f012982ab25bdR31-R46